### PR TITLE
Bugfix/context path

### PR DESCRIPTION
**Pull request by patrickpichler**

### CODE REVIEW

File: cmd/monaco/main.go [hunk 1]
```diff
@@ -69,6 +69,8 @@ We can't wait for your feedback.
 
 	app := cli.NewApp()
 
+	app.ArgsUsage = "[context path]"
```
Comment (didiladi) [inline cmd/monaco/main.go:72]: What's the context path? :thinking: The working directory?

File: cmd/monaco/main.go [hunk 2]
```diff
@@ -136,6 +139,11 @@ Examples:
 	}
 
 	app.Action = func(ctx *cli.Context) error {
+		if ctx.NArg() > 1 {
+			util.Log.Error("Wrong number of arguments! Please provide only the context path or nothing!")
```
Comment (didiladi) [inline cmd/monaco/main.go:143]:
```suggestion
			util.Log.Error("Too many arguments! Either specify a relative path to the working directory, or omit it for using the current working directory.")
```

File: cmd/monaco/main.go [hunk 3]
```diff
@@ -245,6 +254,11 @@ func getDeployCommand(fileReader util.FileReader) cli.Command {
 			},
 		},
 		Action: func(ctx *cli.Context) error {
+			if ctx.NArg() > 1 {
+				util.Log.Error("Wrong number of arguments! Please provide only the context path or nothing!")
```
Comment (didiladi) [inline cmd/monaco/main.go:258]:
```suggestion
				util.Log.Error("Too many arguments! Either specify a relative path to the working directory, or omit it for using the current working directory.")
```

File: pkg/project/project_loader.go [hunk 4]
```diff
@@ -34,7 +34,7 @@ import (
 // if no -p parameter specified, then it creates a list of all projects
 func LoadProjectsToDeploy(specificProjectToDeploy string, apis map[string]api.Api, path string, fileReader util.FileReader) (projectsToDeploy []Project, err error) {
 
-	projectsFolder := filepath.Join(".", path)
+	projectsFolder := filepath.Clean(path)
```
Comment (didiladi) [inline pkg/project/project_loader.go:37]: `Clean` removes the trailing slash?

File: cmd/monaco/main.go (hunk 1 above)
Comment (patrickpichler) [reply cmd/monaco/main.go:72]: Good point ^^

File: pkg/project/project_loader.go (hunk 4 above)
Comment (patrickpichler) [reply pkg/project/project_loader.go:37]: Yes

**Review by didiladi — DISMISSED** [review]

**Comment by didiladi** [comment]
Almost forgot: A test would be nice for using the tool with the trailing slash.

**Comment by patrickpichler** [comment]
This should fix #138

**Comment by patrickpichler** [comment]
There is now a test present to run dry-run with working dir without an slash at the end.

**Review by didiladi — APPROVED** [review]

**Review by didiladi — APPROVED** [review]

**Pull request merged**